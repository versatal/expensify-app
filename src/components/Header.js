import React from 'react';
import { NavLink } from 'react-router-dom'

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <NavLink to="/" activeClassName="isActive" exact={true}>Dashboard</NavLink>
    <NavLink to="/create" activeClassName="isActive">Create Expense</NavLink>
  </header>
)

export default Header;