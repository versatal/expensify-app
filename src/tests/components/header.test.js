import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render header component correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});