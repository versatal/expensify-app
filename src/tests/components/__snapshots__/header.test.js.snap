// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header component correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
</header>
`;
